cmd/cloud-controller-manager/app/controllermanager.go

```diff
@@ -184,10 +184,20 @@ func Run(c *cloudcontrollerconfig.CompletedConfig, stopCh <-chan struct{}) error
 	// add a uniquifier so that two processes on the same host don't accidentally both become active
 	id = id + "_" + string(uuid.NewUUID())
 
```

Comment:
I expected these to be defaulted in config defaulting, not in Run()

Reply:
That makes the default values appear in flag help

Reply:
Sounds good. Shoud we migrate scheduler's option to here for consistent?

Reply:
Fixed